chemprop/utils.py

```diff
@@ -865,7 +867,8 @@ def multitask_mean(
     if metric in scale_dependent_metrics:
         return gmean(scores, axis=axis)
```

Comment:
Could we add in support for this in the geometric means for regression metrics as well?

Reply:
Good idea, I made this change.